src/app/organizations/settings/organization-billing.component.ts

```diff
@@ -5,23 +5,36 @@ import { ApiService } from "jslib-common/abstractions/api.service";
 import { I18nService } from "jslib-common/abstractions/i18n.service";
 import { LogService } from "jslib-common/abstractions/log.service";
 import { PlatformUtilsService } from "jslib-common/abstractions/platformUtils.service";
-
-import { UserBillingComponent } from "../../settings/user-billing.component";
+import { PaymentMethodType } from "jslib-common/enums/paymentMethodType";
+import { TransactionType } from "jslib-common/enums/transactionType";
+import { VerifyBankRequest } from "jslib-common/models/request/verifyBankRequest";
+import { BillingResponse } from "jslib-common/models/response/billingResponse";
 
 @Component({
   selector: "app-org-billing",
-  templateUrl: "../../settings/user-billing.component.html",
+  templateUrl: "./organization-billing.component.html",
 })
-export class OrganizationBillingComponent extends UserBillingComponent implements OnInit {
+export class OrganizationBillingComponent implements OnInit {
+  loading = false;
```

Comment:
Can you add a `// TODO` somewhere in this file, noting that it should be combined with `payment-method.component` when we do the org admin refresh? I think it's reasonable to just copy + paste the logic for now so that we don't go out of scope, but I want to make sure it's reconsidered in the next cycle.

Reply:
I'm not sure I follow - the org admin refresh will follow the same pattern as the current refresh and separate billing/invoice and payment method/balance.

Reply:
Basically, there's a lot of copy + pasted logic duplicated between `organization-billing.component.ts` and `user-billing.component.ts` as a result of these changes. My understanding is that this is temporary to preserve the existing behaviour of the org pages. My thinking is, we want some flag here to make sure this duplicated code doesn't hang around. Otherwise it should be moved to a base class so it can be shared between components.

Reply:
It is only temporary and there will be no duplicated code once the org admin refresh takes place as the `organization-billing.component.ts` will be refactored to extend the `user-billing-history.component.ts` as it exists in production today. Will add a `TODO` as suggested as another reminder!

Reply:
Yes I agree - we're on the same page, just wanted a TODO in there as a reminder, once we've moved on and dumped all of this out of our heads :D